shap/explainers/_tree.py

```diff
@@ -912,6 +912,7 @@ def __init__(self, model, data=None, data_missing=None, model_output=None):
                 "sklearn.ensemble.RandomForestRegressor",
                 "sklearn.ensemble.forest.RandomForestRegressor",
                 "econml.grf._base_grf.BaseGRF",
+                "causalml.inference.tree.CausalRandomForestRegressor",
```

Comment:
is it possible that we add an info that causalml support is experimental using the logging module?

Reply:
Definitely :+1: TreeExplainer class constructor seems to be the best place for it. Is it OK to use warnings module as in the other functions of this module?

Reply:
Added `_safe_check_tree_instance_experimental` for that purpose